libGitWrap/Commit.cpp

```diff
@@ -347,23 +345,23 @@ namespace Git
 
     DiffList Commit::diffFromParent(Result& result, unsigned int index)
     {
-        GW_CD_CHECKED(Commit, DiffList(), result)
+        GW_CD_CHECKED(Commit, DiffList(), result);
```

Comment:
If the compiler didn't complain about this, we should change the macros to either _enforce_ that comma or make it unnecessary (and remove it everywhere).